app/presenters/outcome_presenter.rb

```diff
@@ -31,40 +55,60 @@ def has_calendar?
     calendar.present?
   end
 
-  def has_body?
-    use_template? || super()
-  end
-
   def body
-    if use_template?
-      view_context = @state.dup
```

Comment:
I checked out the commit that introduced `.dup`, removed it, ran all the tests and nothing failed!

I'm happy for it to go particularly as I'm (admittedly crudely) preventing setter methods from being called on the state object from the `ViewContext`. We can always bring it back if/when we realise we're unintentionally changing the state of the `state` object.